app/migration.Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM mhart/alpine-node:12
+ENV NO_UPDATE_NOTIFIER=true
+WORKDIR /opt/app-root/src
+COPY ./ .
+RUN rm -rf /opt/app-root/src/frontend/node_modules \
+ && rm -rf /opt/app-root/src/frontend/dist \
+ && rm -rf /opt/app-root/src/node_modules \
```

Comment:
We should consider using a .dockerignore file to avoid these folders from being grafted from source directory into the image - better approach as it'll avoid having to execute these steps after the fact from the COPY statement.

Reply:
yeah, that helps a lot.  added file